src/fund/constants.ts

```diff
@@ -0,0 +1,4 @@
+// The recommended height of a Coinbase Onramp popup window.
+export const ONRAMP_WIDGET_HEIGHT = 720;
```

Comment:
Can we specify that this is a popup in the naming?

Something like: 

> export const `ONRAMP_POPUP_HEIGHT`

Reply:
Fixed!